src/pybind/mgr/cephadm/tests/fixtures.py

```diff
@@ -55,7 +54,7 @@ def with_cephadm_module(module_options=None, store=None):
             store = {}
         if '_ceph_get/mon_map' not in store:
             m.mock_store_set('_ceph_get', 'mon_map', {
-                'modified': datetime.datetime.utcnow().strftime(CEPH_DATEFMT),
```

Comment:
we're mocking the mon_map here. I mean, if it works, that would be great!

Reply:
The function `datetime_to_str` is using the format `%Y-%m-%dT%H:%M:%S.%fZ`, so everything should be fine.